app/src/main/java/com/sun_asterisk/foodies/screen/main/MainActivity.kt

```diff
@@ -2,12 +2,45 @@ package com.sun_asterisk.foodies.screen.main
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager.widget.ViewPager
 import com.sun_asterisk.foodies.R
+import com.sun_asterisk.foodies.screen.favorite.FavoriteFragment
+import com.sun_asterisk.foodies.screen.home.HomeFragment
+import com.sun_asterisk.foodies.screen.main.layout_adapter.ViewPagerAdapter
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        setupTabLayout()
+    }
+
+    private fun setupViewPager(viewPager: ViewPager) {
+        val adapter = ViewPagerAdapter(supportFragmentManager)
+        adapter.apply {
+            addFragmentContainer(HomeFragment(), resources.getString(R.string.home))
+            addFragmentContainer(FavoriteFragment(), resources.getString(R.string.favorite))
+        }
+        viewPager.adapter = adapter
+    }
+
+    private fun setupTabIcons() {
+        val icons = resources.obtainTypedArray(R.array.layout_icons)
+        for (i in 0..1) {
+            when (i) {
+                0 -> tabs.getTabAt(i).let { it?.setIcon(icons.getDrawable(i)) }
+                else -> tabs.getTabAt(i).let { it?.setIcon(icons.getDrawable(i)) }
+            }
```

Comment:
```suggestion
tabs.getTabAt(MainPage.HOME).let { it?.setIcon(icons.getDrawable(MainPage.HOME)) }
tabs.getTabAt(MainPage.FAVORITE).let { it?.setIcon(icons.getDrawable(MainPage.FAVORITE)) }
```

Và tạo 1 annotation class hoặc tạo biến const val trong MainActivity
``
`@IntDef(MainPage.HOME_PAGE,
        MainPage.FAVORITE_PAGE)

annotation class MainPage {
    companion object {
        const val HOME_PAGE = 0
        const val FAVORITE_PAGE = 1
    }
}`